lib/model/gclEditorModel.js

```diff
@@ -54,7 +54,7 @@ var GCLEditorModel = createClass({
   attrs: {
 
     text: {
-      value: '',
+      value: 'vertex A',
```

Comment:
I can see why this is helpful for testing purposes, but maybe it wasn't meant to make it into the PR.